core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -2418,6 +2418,41 @@ paths:
             - openid
             - email
             - profile
+  '/api/workspaces/{workspaceNamespace}/{workspaceName}/checkSamActionWithLock/{samActionName}':
```

Comment:
I would prefer to not expose the codename "Sam" in API signatures.  Rename this?

Reply:
IAM?